emailproxy.py

```diff
@@ -91,6 +91,16 @@ class NSObject:
             pass
 del no_gui_parser
 
+# by default the proxy stores the OAuth 2.0 tokens in a local config file, but it can optionally store them
+# remotely in AWS Secrets Manager. import the dependency only if this option is specified.
+aws_secrets_parser = argparse.ArgumentParser()
+aws_secrets_parser.add_argument('--aws-secrets', action='store_true')
+
+if aws_secrets_parser.parse_known_args()[0].aws_secrets:
+    import boto3
+    from botocore.exceptions import ClientError
```

Comment:
For consistency I'd `import botocore.exceptions` then use the full package name (not idiomatic Python, but consistent with the rest of the project)